### PR TITLE
Make URL to connect app translatable

### LINKED ISSUE #12263: Need to make 'Connected Apps' link for 'Allow Data Sharing' configurable
Currently the link for 'Allow Data Sharing' under the Connected Apps setting is hardcoded and points to the AUS Discover Regenerative n8n workflow.  If other instances want to use the Connected Apps for their own workflows we need to be able to change the 'Allow Data Sharing' link.  

Connected Apps_Demo Hub_CAN

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

- Closes #12263 

Instance managers can now change the URL in Transifex to use their own API endpoint.

#### What should we test?

- The specs cover this sufficiently.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [x] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by rioug — APPROVED** [review]
Nice one :+1:

File: config/locales/en.yml [hunk 1]
```diff
@@ -266,7 +266,7 @@ en:
   customer_instructions: "Customer instructions"
   additional_information: "Additional Information"
   connect_app:
-    url: "https://n8n.openfoodnetwork.org.uk/webhook/regen/connect-enterprise"
+    url: "https://n8n.openfoodnetwork.org/webhook/regen/connect-enterprise"
```
Comment (dacook) [inline config/locales/en.yml:269]: Why not do this for en_AU with Transifex?

I guess this is quicker, and besides it's now a less-specific URL, so a better example for the default locale :)

Comment (dacook) [reply config/locales/en.yml:269]: Oh, and allows for testing 👍

**Review by dacook — APPROVED** [review]
Great!

**Comment by dacook** [comment]
I guess it's too early to bother with written documentation, but I guess you'll make sure to announce to instance managers (was it Karla?)

**Pull request merged**